дз1ахлюстина/дз1ахлюстина/Program.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace дз1ахлюстина
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            double x;
+            Console.WriteLine("Введите длину ребра октаэдра:");
+            x = double.Parse(Console.ReadLine());
+
+            double S;
+            S = 2 * Math.Pow(x,2) * Math.Sqrt(3);
```

Comment:
Для вычисления квадрата лучше использовать умножение, а не вызов метода Pow
```suggestion
            S = 2 * x * x * Math.Sqrt(3);
```